blueprints/automation/unifiprotect/push_notification_smart_event.yaml

```
@@ -265,7 +265,7 @@ trigger:
 
 condition:
   - "{{ not input_presence_filter or not is_state(input_presence_filter, 'home') }}"
-  - "{{ trigger_object != 'none' and trigger_object != 'unavailable' }}"
+  - "{{ trigger_object != 'none' and trigger_object != 'unavailable' and trigger_object in smart_detect_objs }}"
```

Whoop. I am pretty sure this got messed up when I split them out into 2 blueprints.

You can just change this to this instead:
```suggestion
  - "{{ trigger_object in smart_detect_objs }}"
```